EIPS/eip-1355.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 1355
 title: Ethash 1a
-author: Paweł Bylica (@chfast) <pawel@ethereum.org>, Jean M. Cyr (@jean-m-cyr)
+author: Paweł Bylica (@chfast), Jean M. Cyr (@jean-m-cyr)
```

Comment:
I wonder if we should include the person twice in this case, so that they are a valid author both by email and by GitHub handle?

Reply:
(same comment elsewhere where this change was made)